lxd/lxd.spec

```diff
@@ -332,6 +340,9 @@ getent group %{name} > /dev/null || groupadd -r %{name}
 %doc doc/*
 
 %changelog
+* Tue Apr 27 2021 Link Dupont <link@sub-pop.net> 4.12-0.2
+- Apply patch to configure spice using -spice parameter
+
```

Comment:
Please don't manually update the changelog. Will be generated. Don't worry, you'll get your credit :-)

Reply:
I'm not worried about credit. Mostly this is me stumbling through how to build packages with `tito`. I've never used it before.